src/main/java/thePackmaster/cards/cthulhupack/BeyondTheStars.java

```diff
@@ -13,20 +17,20 @@ public class BeyondTheStars extends AbstractCthulhuCard {
 
     public BeyondTheStars() {
         super(ID, 1, CardType.ATTACK, CardRarity.COMMON, CardTarget.ALL);
-        baseDamage = 6;
-        baseMagicNumber = magicNumber = 2;
+        baseDamage = 4;
         isMultiDamage = true;
-
+        cardsToPreview = new Madness();
     }
 
 
     public void use(AbstractPlayer p, AbstractMonster m) {
         allDmg(AbstractGameAction.AttackEffect.FIRE);
-        Wiz.applyToSelf(new EntropyPower(p, this.magicNumber));
+        AbstractCard c = new Madness();
+        if (upgraded) c.upgrade();
+        AbstractDungeon.actionManager.addToBottom(new MakeTempCardInHandAction(c.makeCopy(), 1));
```

Comment:
Won't break anything, but when you've just made a card using `new`, immediately calling `makeCopy` isn't needed.